packages/backend/src/modules/dashboard/dashboard.repository.ts

```diff
@@ -1,19 +1,58 @@
-import { CreateTaskCommandDTO } from 'shared'
-import DashboardModel from './Dashboard'
+import { CreateTaskCommandDTO, CreateDashboardCommand } from 'shared'
+import mongoose from 'mongoose'
+import { Dashboard as DashboardModel } from './dashboard.model'
 import { Task } from '../task/task'
+import { Dashboard } from './dashboard'
 
 export class DashboardRepository {
-  private readonly _dashboardModel = DashboardModel
+  private readonly _dashboardModel
 
-  constructor(boardModel: any) {
-    this._dashboardModel = boardModel
+  constructor(dashboardModel: any) {
+    this._dashboardModel = dashboardModel
   }
 
-  async addNewTaskToDashboard(createTaskCommand: CreateTaskCommandDTO, savedTask: Task) {
+  async createDashboard(createDashboardCommand: CreateDashboardCommand): Promise<Dashboard> {
+    console.log('Creating new dashboard in DB')
+    const dashboard = await this._dashboardModel.create({
+      _id: new mongoose.Types.ObjectId(),
+      title: createDashboardCommand.title,
+      description: createDashboardCommand.description,
+      imageCoverUrl: createDashboardCommand.imageCoverUrl,
```

Comment:
this Command doesnt has this filed:
TS2339: Property 'imageCoverUrl' does not exist on type 'CreateDashboardCommand'.

Reply:
I have no such problem. Maybe using `yarn libbuild` in the shared package will solve this?

Reply:
```
{
    _id: ObjectId('622e5dd9c3cba876b8800f93'),
    title: 'Tablica 1',
    description: 'Opis tablicy pierwszej',
    createdAt: 'Fri May 06 2022 20:16:40 GMT+0000 (UTC)',
    status: 'PUBLIC',
    users: [
        {
            _id: ObjectId('622e5dffa63863cfc5f812f5'),
            username: 'Marek Kowalski',
            avatarId: 345
        }
    ],
    columns: [
        {
            _id: ObjectId('622e5e084eb135ee631dcd8e'),
            title: 'Kolumna Pierwsza',
            tasks: [
                {
                    _id: ObjectId('622e5e109a5d1fb63a360fb0'),
                    title: 'Tytuł 1',
                    imageCoverId: 'photo-1646388023469-18941b7e3b6f',
                    comments: [
                        {
                            _id: ObjectId('622e5e179c945199e651891a')
                        }
                    ],
                    attachments: [
                        {
                            _id: ObjectId('622e5e1c15be96a29a180abe')
                        }
                    ],
                    users: [
                        {
                            _id: ObjectId('622e5e2094298d670e6129ee')
                        }
                    ]
                }
            ]
        }
    ]
}
```